cmake/qtifw/install_win_copydll.qs

```diff
@@ -38,10 +38,20 @@ function Component()
 
       var regdll = systemTargetDir + "regsvr32.exe";
 
+      console.log("regdll=" + regdll);
+
+      // We'll need "temp" directory extracted from the installer
+      // Based on the @TargetDir@ property, which uses forward slashes by default
+      // Make it use windows native separators
+      var targetDir = installer.value("TargetDir").replace(/\//g, '\\');
+      console.log("targetDir=" + targetDir);
+      var tempDir = targetDir + '\\temp';
+      console.log("tempDir=" + tempDir);
```

Comment:
So there's an extra version of the slash sanitizing.  This doesn't seem like an issue to me, and your comments here really help explain what's going on.